com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml

```diff
@@ -0,0 +1,30 @@
+id: com.github.danrabbit.harvey
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: com.github.danrabbit.harvey
+rename-icon: com.github.danrabbit.harvey
+tags: ['contrast', 'color']
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--talk-name=ca.desrt.dconf'
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  - '--filesystem=xdg-run/dconf'
+  - '--filesystem=~/.config/dconf:ro'
+  - '--own-name=com.github.danrabbit.harvey'
+cleanup:
+  - '*.a'
+  - '*.h'
```

Comment:
I’m pretty sure these only end up in `/include`, so this mightn’t be needed, but I’ll check the build output. Also, if `.a` files get installed, you might want to check if there isn’t something along the lines of `--disable-static` in the build options of the app.